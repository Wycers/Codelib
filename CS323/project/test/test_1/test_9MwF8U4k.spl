int test_1_o03(
{
  int max;
  if (m > n)
  {
    max = m;
  }
  else
  {
    max = n;
  }
  return max;
}
