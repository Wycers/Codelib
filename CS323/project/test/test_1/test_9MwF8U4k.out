Error type B at Line 1: Missing closing parenthesis ')'
